sql/core/src/main/scala/org/apache/spark/sql/execution/exchange/BroadcastExchangeExec.scala

```diff
@@ -73,80 +73,73 @@ case class BroadcastExchangeExec(
 
   @transient
   private[sql] lazy val relationFuture: Future[broadcast.Broadcast[Any]] = {
-    // relationFuture is used in "doExecute". Therefore we can get the execution id correctly here.
```

Comment:
I usually don't like such changes. Wonder if we can keep the indentation same and make it easier to track the history of commits.

Reply:
@HyukjinKwon I have tried to adjust the indent so that diff seems to show only lines i have modified. Does it seem ok now.?

Reply:
The indentation is wrong now although the diff is smaller. how about
```
private[sql] lazy val relationFuture: Future[broadcast.Broadcast[Any]] = {
  SQLExecution.withThreadLocalCaptured ... {
    doBroadcast(..)
  }
}

private def doBroadcast ... {
  // original code
}
```

Reply:
@cloud-fan updated as per your suggestion, but still it changes indent than original. Is it ok.?

Reply:
ah sorry I misread the code. Seems we can't avoid changing the indentation as it was so nested before. I'm OK with your original code.

Reply:
reverted